analyzers/src/SonarAnalyzer.Common/Common/AnalyzerConfiguration.cs

```diff
@@ -41,10 +41,20 @@ public class AnalyzerConfiguration
             new HotspotConfiguration(new RuleLoader());
 
         public static IAnalyzerConfiguration AlwaysEnabled { get; } =
-            new AlwaysEnabledConfiguration();
+            new AlwaysEnabledConfiguration(false);
+
```

Comment:
There were few improvements suggested on the other PR. I'm not sure which PR will make it first to be merged.
This is just a placeholder to resolve in the future.

Reply:
Fixed it in that PR. Leaving it untouched here to not produce conflicts for now

Reply:
Rebased